-- Fix build with the setuptools version available in FreeBSD ports.
-- Upstream requires setuptools>=82.0.1; lower the requirement to >=61
-- which is what the port provides.
-- Also remove the SPDX 'license' and 'license-files' fields that the older
-- setuptools rejects, replace the packages list with a packages.find include,
-- and use a wildcard package-data key to avoid the hyphenated package name
-- that older setuptools rejects in [tool.setuptools.package-data].

--- pyproject.toml.orig	2026-10-10 20:44:59 UTC
+++ pyproject.toml
@@ -1,12 +1,10 @@ build-backend = "setuptools.build_meta"
 [build-system]
 build-backend = "setuptools.build_meta"
-requires = ["setuptools>=82.0.1"]
+requires = ["setuptools>=61"]
 
 [project]
 name = "types-pytz"
 version = "2026.5.0.20261006"
-license = "Apache-2.0"
-license-files = ["LICENSE"]
 description = "Typing stubs for pytz"
 readme = { text = """\
 ## Typing stubs for pytz
@@ -50,8 +48,10 @@ dependencies = []
 "Chat" = "https://gitter.im/python/typing"
 
 [tool.setuptools]
-packages = ['pytz-stubs']
 include-package-data = false
 
+[tool.setuptools.packages.find]
+include = ["pytz-stubs"]
+
 [tool.setuptools.package-data]
-'pytz-stubs' = ['__init__.pyi', 'exceptions.pyi', 'lazy.pyi', 'reference.pyi', 'tzfile.pyi', 'tzinfo.pyi', 'METADATA.toml', 'py.typed']
+'*' = ['__init__.pyi', 'exceptions.pyi', 'lazy.pyi', 'reference.pyi', 'tzfile.pyi', 'tzinfo.pyi', 'METADATA.toml', 'py.typed']
